tests/configs/config_test_driver.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Mbed TLS configuration for PSA test driver libraries. It includes:
```

Comment:
I assume that moving forward there will be two versions of the test driver?

- `tests/configs/config_test_driver.h` for 3.6
- `tf-psa-crypto/tests/configs/config_test_driver.h` for devel, and 4.0 onwards? 

Would it be good to add a comment to link those, so when someone is adding a feature they will know about it to backport it here?